doc/topics/jinja/index.rst

```diff
@@ -1473,6 +1476,15 @@ Returns:
 File filters
 ------------
 
+.. jinja_ref:: connection_check
+
+``connection_check``
+-------------
```

Comment:
Can you make the `___` line match the number of characters in the line above? Otherwise a warning will be generated in the doc build.